conf/nomad/presto_standalone.hcl

```diff
@@ -40,24 +40,32 @@ job "${nomad_job_name}" {
             }
           }
         }
+        sidecar_task {
+          driver = "docker"
+          resources {
+            cpu    = "${cpu_proxy}"
+            memory = "${memory_proxy}"
+          }
+        }
       }
-      check {
-        task     = "server"
-        name     = "presto-hive-availability"
-        type     = "script"
-        command  = "presto"
-        args     = ["--execute", "SHOW TABLES IN hive.default"]
-        interval = "30s"
-        timeout  = "15s"
-      }
-      check {
-        expose   = true
-        name     = "presto-info"
-        type     = "http"
-        path     = "/v1/info"
-        interval = "10s"
-        timeout  = "2s"
-      }
+//      TODO: Checks needs to be implemented in another way due to sidecar
```

Comment:
Could you elaborate on it?

Reply:
Sorry, not really.  If you uncomment and run the module you get an error that mentions that the `check` needs to be implemented using a proxy, otherwise it's not allowed to communicate. At least something along those lines.

Reply:
The first check doesn't seem to be using expose? https://www.nomadproject.io/docs/job-specification/expose

Reply:
It has something to do with the `sidecar_task` i think. When I added that, it failed on both `check` stanzas.